packages/ckb-sdk-core/examples/sendTransaction.js

```diff
@@ -10,20 +10,31 @@ const bootstrap = async () => {
 
   const secp256k1Dep = await core.loadSecp256k1Dep() // load the dependencies of secp256k1 algorithm which is used to verify the signature in transaction's witnesses.
 
+  const publicKey = core.utils.privateKeyToPublicKey(privateKey)
   /**
-   * genereat address object, who has peroperties like private key, public key, sign method and verify mehtod
-   * - value, the address string
-   * - privateKey, the private key in hex string format
-   * - publicKey, the public key in hex string format
-   * - publicKeyHash, the publicKeyHash of the public key, a blake160-ed public key is use here
-   * - sign(msg): signature string
-   * - verify(msg, signature): boolean
+   * to see the public key
    */
-  const myAddressObj = core.generateAddress(privateKey)
+  // console.log(`Public key: ${publicKey}`)
+
+  const publicKeyHash = `0x${core.utils.blake160(publicKey, 'hex')}`
   /**
-   * to see the address
+   * to see the public key hash
    */
-  console.log(myAddressObj.value)
+  // console.log(`Public key hash: ${publicKeyHash}`)
```

Comment:
useless annotation?

Reply:
It's for someone who is unfamiliar with JavaScript.